### DISCUSSION TITLE
如何在 footer 内放入备案链接跳转

### DISCUSSION THREAD

**Discussion by watsonhaw5566** [web-infra-dev/rspress#1045] [Q&A]
感谢 rspress 团队提供这么优秀的文档工具。我司目前遇到的问题是由于文档想暴露在公网中使用，此时就需要在底部 footer 的位置放置对应的备案号和<a>标签的链接跳转，方可正常备案。我翻看了一下 rspress 的文档发现当前 footer 只能配置 [message](https://rspress.dev/zh/api/config/config-theme#footer)，所以想请教一下贵团队可否有对应的解决方案，亦或一些“小技巧”，感谢。

**Comment by SoonIter** [comment]
感谢使用，这一块好像确实缺失了，可以直接pr贡献，或者尝试一下html.tag插一段内联js解下燃眉之急

**Reply by watsonhaw5566** [reply]
貌似目前只能通过自定义实现，感谢。

**Comment by xc2** [comment]
临时解决方案：使用自定义主题。可以参考：

https://github.com/rspack-contrib/rspack-examples/pull/59/files#diff-df52f2c335f5c9c12c2c98a462aae7cdd118c7902266a49a6c78701a2b9c7486

**Reply by watsonhaw5566** [reply]
感谢您的回复。

**Reply by xc2** [reply]
客气了
